build.jam

```diff
@@ -0,0 +1,22 @@
+# Copyright René Ferdinand Rivera Morell 2023
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+project /boost/variant2
+    : common-requirements
```

Comment:
Why are these project requirements and not requirements of `boost_variant2`?

Reply:
I most certainly don't want these to apply to all subprojects, e.g. `doc`. They have nothing to do with the subprojects.